### PR TITLE
33 create actionpanel

### PR DESCRIPTION
**Pull request by haakonjacobsen**
Added a lot of stuff.

* Created ActionPanel
* Created Planning Defence and Planning Attack within Action Panel
* Created UnitCounterPanel
* Removed TowerType type to UnitType (FIRE, WATER, GRASS, NONE)
* Fixed bug for starting the game (related to UnitType)

### CODE REVIEW

File: client/core/src/com/cows/game/enums/TowerType.kt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package com.cows.game.enums
```
Comment (AndreasWintherMoen) [inline client/core/src/com/cows/game/enums/TowerType.kt:1]: Can this file be removed?

File: client/core/src/com/cows/game/hud/PlanningActionPanel.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import com.cows.game.Application
+
+
+abstract class PlanningActionPanel(): ActionPanel() {
+    companion object {
+        val counterPanelHeight = getPercentOfScreen(15f, Application.HEIGHT)
+        val buttonPanelHeight = getPercentOfScreen(15f, Application.HEIGHT)
+
+        private fun getPercentOfScreen(percentage: Float, screenWidth: Float): Any {
```
Comment (AndreasWintherMoen) [inline client/core/src/com/cows/game/hud/PlanningActionPanel.kt:11]: Make return type float (or double)

**Review by AndreasWintherMoen — CHANGES_REQUESTED** [review]
Nice! Some minor fixes requested. I've ignored the println statements, but before final delivery we should make sure we remove all unnecessary printing.

File: client/core/src/com/cows/game/hud/PlanningDefenseActionPanel.kt [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.g2d.SpriteBatch
+import com.badlogic.gdx.math.Vector2
+import com.cows.game.enums.TowerType
+import com.cows.game.enums.UnitType
+import com.cows.game.managers.FunctionDelayer
+import com.cows.game.managers.TowerSpawner
+
+class PlanningDefenseActionPanel(private val onStartGame: () -> Unit): PlanningActionPanel() {
+    //val panelBatch = SpriteBatch()
```
Comment (blauks) [inline client/core/src/com/cows/game/hud/PlanningDefenseActionPanel.kt:12]:
```suggestion
```

**Review by AndreasWintherMoen — APPROVED** [review]
Neat

**Pull request merged**